api/config.py

```diff
@@ -151,6 +164,18 @@ class Configuration(CoreConfiguration):
             "key": HELP_URI,
             "label": _("Patron support custom integration URI")
         },
+        {
+            "key": LARGE_COLLECTION_LANGUAGES,
+            "label": _("The primary languages represented in this library's collection")
+        },
+        {
+            "key": SMALL_COLLECTION_LANGUAGES,
+            "label": _("Other major languages represented in this library's collection")
+        },        
+        {
+            "key": TINY_COLLECTION_LANGUAGES,
+            "label": _("Other languages in this library's collection")
+        },        
```

Comment:
There's "type": "select" which has fixed options and "type": "list" where the UI lets you add your own things. The only other thing "type": "list" is used for is the allowed domains for admin authentication with google. Type "list" seems like the best option for now.